src/java.base/share/classes/java/lang/classfile/constantpool/ConstantPoolBuilder.java

```diff
@@ -365,7 +365,7 @@ default MethodHandleEntry methodHandleEntry(DirectMethodHandleDesc descriptor) {
      * it is returned; otherwise, a new entry is added and the new entry is
      * returned.
      *
-     * @param refKind the reference kind of the method handle {@jvms 4.4.8}
+     * @param refKind the reference kind of the method handle JVMS {@jvms 4.4.8}
```

Comment:
Same, these occurrences are not part of a sentence, so enclose in parentheses like `(JVMS {@jvms 4.4.8})`